cmd/prepare_worker.go

```diff
@@ -186,6 +187,22 @@ func (b *workerBuilder) buildTypeScript(ctx context.Context, baseDir string) (sd
 			return nil, fmt.Errorf("version not found in package.json")
 		}
 	}
+
+	// Prep the TypeScript runner to be built
+	cmd := exec.Command("npm", "install")
+	cmd.Dir = "./workers/typescript"
+	err := cmd.Run()
+	if err != nil {
+		return nil, fmt.Errorf("npm install in ./workers/typescript failed: %w", err)
+	}
+
+	cmd = exec.Command("npm", "run", "build")
+	cmd.Dir = "./workers/typescript"
+	err = cmd.Run()
+	if err != nil {
+		return nil, fmt.Errorf("npm run build in ./workers/typescript failed: %w", err)
+	}
```

Comment:
I don't think this part can really move in there since it's about prepping the worker rather than the SDK itself or the generated project.

As far as the more general thing goes, yeah, I think it's just nice to have this in here to make life easier for devs using the project for the first time. I was wondering though if maybe it might work nicely for us to move to something like Bazel or Buck or whatever to deal with all this building rather than writing it ourselves. That said, what we have is still pretty small and that might be a big undertaking. :shrug:

Reply:
Agree with Spencer, this is to make it easier for devs running Omes manually directly out of the repo—it fixes a pain point that caused me to lose an hour or so and that I had to ask someone about. Putting it in features wouldn't help.

Reply:
> Putting it in features wouldn't help.

Do they not also need to have the pain point fixed of running build on the project?